feg/cloud/go/services/feg/obsidian/models/swagger.v1.yml

```diff
@@ -345,7 +345,7 @@ paths:
 
   /feg/{network_id}/gateways:
     get:
-      summary: List all federation gateways for a federation network
+      summary: List all gateways for a federation network
```

Comment:
Revert this change - our definitions of `federation network`, `federated lte network`, `lte network`, etc. are a bit confusing, and this additional context helps.

Reply:
the reason i have this is to adhere to the `POST` register gateway naming for all the other networks
```
                    - “List all gateways for a Wifi network”
                    - “List all gateways for a carrier wifi network”
                    - “List all federation gateways for a federation network” [ RENAME ]
                    - “List all gateways for a network”
                    - “List all gateways for an LTE network”
  ```